spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -80,32 +103,57 @@ definitions:
             type: CarrierPhase
             units: cycles
             desc: Carrier phase observation with typical sign convention.
+        - D:
+            type: Doppler
+            units: Hz
+            desc: Doppler observation with typical sign convention.
         - cn0:
             type: u8
             units: dB Hz * 4
             desc: Carrier-to-Noise density
+        - snr:
+            type: u8
+            units: dB * 4 - 20
+            desc: Signal-to-Noise ratio. A fixed offset of -20 dB is applied.
         - lock:
             type: u16
             desc: |
               Lock indicator. This value changes whenever a satellite
               signal has lost and regained lock, indicating that the
               carrier phase ambiguity may have changed.
+        - flags:
+            type: u16
+            desc: |
+              Measurement status flags. A bit field of flags providing the
+              status of this obervation.
+            fields:
+              - 2-15:
```

Comment:
Oh yeah, I was going to add the rest of the flags, once I had nailed down exactly how I was going to represent them in the firmware.

And I did not know about the proposal last night when I wrote this.